deploy.sh

```diff
@@ -44,7 +44,11 @@ cd $DEST_DIR
 # Transform the readme
 if [ -f readme.md ]; then
 	mv readme.md readme.txt
-	sed -i '' -e 's/^# \(.*\)$/=== \1 ===/' -e 's/ #* ===$/ ===/' -e 's/^## \(.*\)$/== \1 ==/' -e 's/ #* ==$/ ==/' -e 's/^### \(.*\)$/= \1 =/' -e 's/ #* =$/ =/' readme.txt
+	if [ -f CHANGELOG.md ]; then
+		cat CHANGELOG.md >> readme.txt
+		rm CHANGELOG.md
+	fi
+	sed -i '' -e 's/^# \(.*\)$/=== \1 ===/' -e 's/ #* ===$/ ===/' -e 's/^## \(.*\)$/== \1 ==/' -e 's/ #* ==$/ ==/' -e 's/^### \(.*\)$/= \1 =/' -e 's/ #* =$/ =/' -e 's/\*\*//g' readme.txt
```

Comment:
@twoelevenjay I'm not sure I understand this change. WP.org treats `**` the same way as Markdown and converts them into **strong**. Why remove that formatting?